README.md

```diff
@@ -1151,6 +1153,8 @@ Future.both(a, b).fork(console.error, console.log);
 //> ['a', 'b']
 ```
 
+When one Future resolves, the other gets cancelled automatically.
```

Comment:
This should be *rejects*, rather than *resolves*. When one of the Futures resolves, we wait for the other one to do too, because we are actually awaiting the result of both. :)